Policies/github-terms-of-service.md

```diff
@@ -127,45 +128,58 @@ You agree not to engage in activity that significantly harms our Users. We will
+You retain all moral rights to Your Content that you upload, publish, or submit to any part of the Service, including the rights of integrity and attribution. However, you waive these rights and agree not to assert them against us, to enable us to reasonably exercise the rights granted in Section D.4, but not otherwise.
 
-To the extent such an agreement is not enforceable by applicable law, you grant GitHub a nonexclusive, revocable, worldwide, royalty-free right to (1) use the Content without attribution strictly as necessary to render the Website and provide the Service; and (2) make reasonable adaptations of the Content as provided in this Section. We need these rights to allow basic functions like search to work. 
+To the extent this agreement is not enforceable by applicable law, you grant GitHub the rights we need to use Your Content without attribution and to make reasonable adaptations of Your Content as necessary to render the Website and provide the Service.
 
-#### 8. Private Repositories
-Paid accounts may have [private repositories](/articles/making-a-public-repository-private/), which allow the User to control access to Content. GitHub employees [only access private repositories when access is required for security or maintenance](/articles/github-security/#employee-access) or for support reasons, and then only with the consent of the repository owner. Because we respect your privacy in private repositories, you must respect the Terms of Service in everything you post to a private repository. If we have reason to believe the contents of a private repository are in violation of the law or of these Terms, we have the right to remove them.
+### E. Private Repositories
+**Short version:** *You may have access to private repositories. We treat the content of private repositories as confidential, and we only access it for support reasons, with your consent, or if required to for security reasons.*
 
-### E. Copyright Infringement and DMCA Policy
+#### 1. Control of Private Repositories.
+Paid accounts may have private repositories, which allow the User to control access to Content.
+#### 2. Confidentiality of Private Repositories.
+GitHub considers the contents of private repositories, and only the contents of private repositories, to be confidential to you. GitHub will protect the contents of private repositories from unauthorized use, access, or disclosure in the same manner that we would use to protect our own confidential information of a similar nature and in no event with less than a reasonable degree of care.
+#### 3. Access.
+GitHub employees may only access the content of your private repositories in the following situations:
+- With your consent and knowledge, for support reasons. If GitHub accesses a private repository for support reasons, we will only do so with the owner’s consent and knowledge.
+- When access is required for security reasons.
```

Comment:
Thanks, everyone, for your suggestions! 

We really worked long and hard on how best to put more language around what we mean by "security reasons." After a great deal of discussion, we think we already give enough context in our Terms of Service, and in the event of a security incident that puts GitHub at risk, we need to make sure we are able to respond quickly.

More specifically, one thing to understand here is that we have intentionally restricted ourselves a _lot_ when it comes to accessing private repositories. We've given our users not just a privacy clause but a full confidentiality agreement, and our agreement establishes that by default, GitHub employees may _not_ access private repository contents. There are only certain enumerated situations in which we can access private repository contents, and we limit the number of employees with the ability to do so. 

For those asking, our internal security investigations are _completely_ different from our responses to warrants, subpoenas, and other court actions. Section E.4 of the Terms of Service does reference our [policy on compelled disclosure](https://github.com/github/site-policy/blob/388f344f4e30a2840e6d27d2854e05d7e8e5e287/articles/github-privacy-statement/#how-we-respond-to-compelled-disclosure).

So for right now, we're going to leave this as it is. However, if we get more discussion about it in our next comment period, we're certainly willing to reconsider.